service/src/main/java/bio/terra/drshub/config/DrsProviderInterface.java

```diff
@@ -58,37 +58,6 @@ default List<AccessMethodConfigTypeEnum> getAccessMethodConfigTypes() {
     return getAccessMethodConfigs().stream().map(ProviderAccessMethodConfig::getType).toList();
   }
 
-  /**
```

Comment:
This block and the next were unused in production code. A bunch of tests went away with this too.